x/stakeibc/keeper/unbonding_records.go

```diff
@@ -267,7 +267,16 @@ func (k Keeper) UnbondFromHostZone(ctx sdk.Context, hostZone types.HostZone) err
 	// Determine the ideal balanced delegation for each validator after the unbonding
 	//   (as if we were to unbond and then rebalance)
 	// This will serve as the starting point for determining how much to unbond each validator
-	delegationAfterUnbonding := hostZone.TotalDelegations.Sub(totalUnbondAmount)
+	// first, get the total delegations _excluding_ validators with slash_query_in_progress
+	totalValidDelegationBeforeUnbonding := sdkmath.ZeroInt()
+	for _, validator := range hostZone.Validators {
+		if !validator.SlashQueryInProgress {
+			totalValidDelegationBeforeUnbonding = totalValidDelegationBeforeUnbonding.Add(validator.Delegation)
+		}
+	}
+	// then subtract out the amount to unbond
+	delegationAfterUnbonding := totalValidDelegationBeforeUnbonding.Sub(totalUnbondAmount)
```

Comment:
should we add a check here that `delegationAfterUnbonding` is not negative? possible it's already checked downstream

Reply:
I think it's checked in the next function (`k.GetTargetValAmtsForHostZone`) but happy to add an additional check if you think it would make things more clear / has better code hygiene!